composer.json

```
@@ -21,7 +21,7 @@
     }
   },
   "require": {
-    "sabberworm/php-css-parser": "8.1.*"
+    "sabberworm/php-css-parser": "8.3.*"
```

This should allow allow non-breaking updates in the future too, like so:

```suggestion
    "sabberworm/php-css-parser": "^8.3"
```